main.go

```diff
@@ -166,6 +167,18 @@ func main() {
 							&cli.StringFlag{Name: "o", Usage: "Output folder", Required: false},
 						},
 					},
+					{
+						Name:  LANG_TYPESCRIPT,
+						Usage: "Generate an SDK for use with TypeScript",
```

Comment:
This PR is the first that adds off chain SDK generation. Having said that we now clearly differentiate what kind of SDK is being generated (on-chain or off-chain). Looking at this help text doesn't differentiate that
```
LANGUAGES:
   go          Generate an SDK for use with Golang
   python      Generate an SDK for use with Python
   java        Generate an SDK for use with Java
   csharp      Generate an SDK for use with C#
   typescript  Generate an SDK for use with TypeScript
```
My suggestion for this is something like
```
Generate an off-chain SDK for use with TypeScript
```
and then for the other languages to add "on-chain". Once there is a language that has both on-chain and off-chain support we can make the help text for that language generic again and add an option flag for the user to choose between types.